### PR TITLE
Add `frustumContainsPoint` to  `CollisionUtil`

### PR DESCRIPTION
**Pull request by HypnosNova**
Plane: distanceToPoint
Frustum: containsPoint

### CODE REVIEW

File: packages/math/src/BoundingFrustum.ts [hunk 1]
```diff
@@ -143,6 +144,33 @@ export class BoundingFrustum implements IClone<BoundingFrustum>, ICopy<BoundingF
...
+      return false;
+    }
+    if (Plane.distanceToPoint(this.near, point) < 0) {
+      return false;
+    }
+    if (Plane.distanceToPoint(this.far, point) < 0) {
+      return false;
+    }
+    return true;
+  }
```
Comment (GuoLei1990) [inline packages/math/src/BoundingFrustum.ts:172]: Add this in `CollisionUtil` and named `frustumContainspoint`, all determining intersections, containment, etc fun is in `CollisionUtil`

Comment (HypnosNova) [reply packages/math/src/BoundingFrustum.ts]: handled

File: packages/math/src/Plane.ts [hunk 2]
```diff
@@ -53,6 +53,15 @@ export class Plane implements IClone<Plane>, ICopy<Plane, Plane> {
...
   }
 
+  /**
+   * Calculate the distance from a plane to a point.
+   * @param plane - The plane
+   * @param point - The point
+   */
+  static distanceToPoint(plane: Plane, point: Vector3) {
+    return Vector3.dot(plane.normal, point) + plane.distance;
+  }
```
[the remark this answers is not in the document]
Comment (HypnosNova) [reply packages/math/src/Plane.ts]: handled

**Review by singlecoder — APPROVED** [review]
+1

File: packages/math/src/CollisionUtil.ts [hunk 3]
```diff
@@ -361,6 +361,52 @@ export class CollisionUtil {
...
+      return ContainmentType.Disjoint;
+    }
+    distance = CollisionUtil.distancePlaneAndPoint(frustum.far, point);
+    if (Math.abs(distance) <= MathUtil.zeroTolerance) {
+      return ContainmentType.Intersects;
+    } else if (distance < 0) {
+      return ContainmentType.Disjoint;
+    }
+    return ContainmentType.Contains;
+  }
```
Comment (GuoLei1990) [inline packages/math/src/CollisionUtil.ts:408]: suggeset order is: 
1.near
2.far
3.left
4.right
5.top
6.bottom

**Review by GuoLei1990 — APPROVED** [review]

**Pull request merged**